packages/dso-toolkit/src/styles/components/_breadcrumbs.scss

```diff
@@ -10,13 +10,15 @@ ol.breadcrumb {
     float: left;
 
     + li::before {
-      color: $grijs-60;
-      content: "/";
+      background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' id='Layer_1' x='0' y='0' version='1.1' viewBox='0 0 21.1 55.7' xml:space='preserve'%3E%3Cpath fill='%23666' d='M0 45.6c0-.1.1-.4.4-1.1L14.9 8.6c.2-.5.5-.8 1-1 .5-.2 1.1-.3 1.9-.3h.7c1.8 0 2.6.3 2.6.8 0 .2-.1.6-.4 1.1l-14.5 36c-.2.5-.5.8-1 1-.4.2-1.1.2-1.9.2h-.8c-1.7 0-2.5-.2-2.5-.8z'/%3E%3C/svg%3E");
```

Comment:
Het attribuut `id` is niet nodig, en ik denk `x` en `y` ook (default values zijn toch al 0?). En ik denk zelfs dat je het `xml:space` attribuut ook niet nodig hebt, je schrijft namelijk geen tekst. Kan je hier nog naar kijken?

Reply:
sterker nog, ik lees dat die `xml:space` deprecated is: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xml:space

het `id` en de `x` en `y` heb ik verwijderd